Makefile

```diff
@@ -156,25 +156,37 @@ keystore: export KEYSTORE_PATH ?= $(HOME)/.gradle/status-im.keystore
 keystore: ##@prepare Generate a Keystore for signing Android APKs
 	@./scripts/generate-keystore.sh
 
+fdroid-nix-dir: ##@prepare Create /nix directory for F-Droid Vagrant builders
+	mkdir -m 0755 /nix
+	chown vagrant /nix
+
 #----------------
 # Release builds
 #----------------
-release: release-android release-ios ##@build build release for Android and iOS
+release: release-android release-ios ##@build Build release for Android and iOS
 
-release-android: export TARGET := default
 release-android: export BUILD_ENV ?= prod
 release-android: export BUILD_TYPE ?= nightly
 release-android: export BUILD_NUMBER ?= $(TMP_BUILD_NUMBER)
 release-android: export KEYSTORE_PATH ?= $(HOME)/.gradle/status-im.keystore
 release-android: export ANDROID_APK_SIGNED ?= true
 release-android: export ANDROID_ABI_SPLIT ?= false
 release-android: export ANDROID_ABI_INCLUDE ?= armeabi-v7a;arm64-v8a;x86
-release-android: keystore ##@build build release for Android
+release-android: keystore ##@build Build release for Android
+	scripts/release-android.sh
+
+release-fdroid: export BUILD_ENV ?= prod
+release-fdroid: export BUILD_TYPE = release
+release-fdroid: export ANDROID_APK_SIGNED = false
+release-fdroid: export ANDROID_ABI_SPLIT ?= false
+release-fdroid: export ANDROID_ABI_INCLUDE = armeabi-v7a;arm64-v8a;x86;x86_64
+release-fdroid: ##@build Build release for F-Droid
+	echo "GOOGLE_FREE=1" >> .env.release
 	scripts/release-android.sh
```

Comment:
I don't think I like this `echo "GOOGLE_FREE=1" >> .env.release` in here. It seems clunky.

Reply:
I don't think we need it at the moment, we completely remove the dependency on the other branch.
According to the developer, if we upgrade to 8 rn-device info we would not need it. 
but I'd have to look more into it

Reply:
What about this then:
https://github.com/status-im/status-react/blob/f71faee5c2a869485e9dfafe3de5e704838a6ca2/src/status_im/utils/config.cljs#L47
https://github.com/status-im/status-react/blob/f71faee5c2a869485e9dfafe3de5e704838a6ca2/src/status_im/acquisition/core.cljs#L111-L123
Does it still matter? Or should it be removed?

Reply:
And I'd assume this:
https://github.com/status-im/status-react/blob/f71faee5c2a869485e9dfafe3de5e704838a6ca2/react-native.config.js#L1
And this:
https://github.com/status-im/status-react/blob/f71faee5c2a869485e9dfafe3de5e704838a6ca2/android/app/build.gradle#L129
Would also be obsolete?

Reply:
I'm gonna get rid of the line in `status-react/android/app/build.gradle` first since it seems to do nothing.